dashboard/config/application.rb

```diff
@@ -60,6 +61,8 @@ class Application < Rails::Application
     config.middleware.insert_after SharedResources, NetSimApi
     config.middleware.insert_after NetSimApi, AnimationLibraryApi
     config.middleware.insert_after AnimationLibraryApi, SoundLibraryApi
+    config.middleware.insert_after SoundLibraryApi, DatasetLibraryApi
```

Comment:
Not blocking, question for my own learning purposes: do the Apis have to be listed in the order that the Apis were created (which I assume is how it's listed right now) or does the middleware not care about the order?

Reply:
The order of the APIs here determines the order in which the system will check to see which pattern the routes match against. so if an earlier route matched the same request, then you'd see that it would never reach the API later in the list.